### PR TITLE
update sane-reports-pdf

### PR DESCRIPTION
**Pull request by yanyo87**
## Status
In Progress

## Related Issues
Related: [link to the issue](https://github.com/demisto/sane-reports/pull/222)

## Description
allow page brakes inside threat intel reports 

Fixes CRTX-57370
https://jira-hq.paloaltonetworks.local/browse/CRTX-57370

### CODE REVIEW

**Review by h88 — APPROVED** [review]

**Review by avishbran — APPROVED** [review]

File: docker/sane-pdf-reports/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Last modified: Wed, 23 Mar 2022 12:00:00 +0200
+# Last modified: Tue, 28 Jun 2022 10:41:00 +0200
 FROM demisto/python3-deb:3.9.6.22912
```
Comment (wissamg) [inline docker/sane-pdf-reports/Dockerfile:2]:
```suggestion
FROM demisto/python3-deb:3.10.4.31559
```

Comment (wissamg) [reply docker/sane-pdf-reports/Dockerfile]: then you need to update the Pipfile to use :
python_version = "3.10"

Comment (yanyo87) [reply docker/sane-pdf-reports/Dockerfile]: the main Pipfile ? 
image
cause i don't have one in the 'sane-pdf-reports' folder 
image

Comment (wissamg) [reply docker/sane-pdf-reports/Dockerfile]: Aha I see I thought you were doing the sane-doc-reports
in that case you are good but still I need to figure out why the build is not running. (I'm checking)

Comment (yanyo87) [reply docker/sane-pdf-reports/Dockerfile]: > Aha I see I thought you were doing the sane-doc-reports in that case you are good but still I need to figure out why the build is not running. (I'm checking)

thanks !

**Pull request closed without merging**